src/CoreText/CTFontManager.cs

```diff
@@ -68,18 +95,31 @@ public partial class CTFontManager {
 		[return: MarshalAs (UnmanagedType.I1)]
 		static extern bool CTFontManagerIsSupportedFont (IntPtr url);
 
+#if !NET
 		[Deprecated (PlatformName.MacOSX, 10, 6)]
 		[Unavailable (PlatformName.iOS)]
+#else
+		[UnsupportedOSPlatform ("ios")]
+		[UnsupportedOSPlatform ("macos10.6")]
+#endif
 		public static bool IsFontSupported (NSUrl url)
 		{
 			if (url == null)
 				throw new ArgumentNullException ("url");
 			return CTFontManagerIsSupportedFont (url.Handle);
 		}
 #elif !XAMCORE_3_0
+#if !NET
 		[Obsolete ("API not available on iOS, it will always return false.")]
 		[Deprecated (PlatformName.MacOSX, 10, 6)]
 		[Unavailable (PlatformName.iOS)]
+#else
+		[UnsupportedOSPlatform ("ios")]
+		[UnsupportedOSPlatform ("macos10.6")]
+#if MONOMAC
+		[Obsolete ("Starting with macos10.6 API not available on iOS, it will always return false.", DiagnosticId = "BI1234", UrlFormat = "https://github.com/xamarin/xamarin-macios/wiki/Obsolete")]
+#endif
```

Comment:
on iOS but we have a #if MONOMAC is that correct?

Reply:
The old message was wrong since the API always return `false`, even if macOS was lower than 10.6 (as we're not calling anything)

Reply:
Should I change the obsolete message in dotnet to be 
"Starting with macos10.6 API will always return false."
?

Reply:
The API always `return false` irrespective of the OS or platform
Make this a single, OS-less and version-less, `[Obsolete]`